auditbeat/module/auditd/audit_linux.go

```diff
@@ -251,6 +300,15 @@ func (ms *MetricSet) receiveEvents(done <-chan struct{}) (<-chan []*auparse.Audi
 			if err != nil {
 				continue
 			}
+			if raw.Type == auparse.AUDIT_GET {
+				status := &libaudit.AuditStatus{}
+				if err := status.FromWireFormat([]byte(raw.Data)); err == nil {
+					statusC <- status
```

Comment:
If for some reason `statusC` is not being drained do we want to block the reader loop? Perhaps this should be a non-blocking channel send that drops status events if the channel is full.